portal-kernel/src/com/liferay/portal/kernel/service/packageinfo

```diff
@@ -1 +1 @@
-version 22.2.0
\ No newline at end of file
+version 22.1.0
```

Comment:
As you haven't modified the kernel, I think you don't need to include this commit, as I guess it is an error that will be fixed by Brian in some point. Probably he merged some code that didn't contain the proper baseline commit

Reply:
Ok, good to know. I only did it becouse the jenkins report.